packages/adminrouter/extra/src/test-harness/tests/test_agent.py

```diff
@@ -0,0 +1 @@
+# Copyright (C) Mesosphere, Inc. See LICENSE file for details.
```

Comment:
The file was not empty in master: https://github.com/dcos/dcos/blob/master/packages/adminrouter/extra/src/test-harness/tests/test_agent.py I am going to accept the master version for this merge-conflict.